dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -164,4 +165,95 @@ def test_case_sensitivity(self):
             expected_word_count = {'hello': 1, 'name': 1, 'grant': 2, 'bob': 1,
                                    'friends': 1}
             self.assertDictEqual(expected_word_count, profile['word_count'])
-        
\ No newline at end of file
+
+    def test_text_profile_with_wrong_options(self):
+        with self.assertRaisesRegex(ValueError,
+                "TextProfiler parameter 'options' must be of type"
+                " TextProfilerOptions."):
+            TextProfiler("Name", options="wrong_data_type")
+
+    def test_options_default(self):
+        options = TextProfilerOptions()
+
+        text_profile = TextProfiler("Name", options=options)
+        sample = pd.Series(["This is test, a Test sentence.!!!"])
+        text_profile.update(sample)
+        profile = text_profile.profile
+
+        expected_word_count = {'sentence': 1, 'Test': 1, 'test': 1}
+        expected_vocab = [' ', ',', '.', '!', 'T', 'h', 'i', 's', 'a',
+                          'e', 't', 'n', 'c']
+        self.assertDictEqual(expected_word_count, profile['word_count'])
+        self.assertCountEqual(expected_vocab, profile['vocab'])
+
+    def test_options_case_sensitive(self):
+        options = TextProfilerOptions()
+        options.is_case_sensitive = False
```

Comment:
should add a comment saying we are assuming the default is tested in previous tests

Reply:
added